omou-frontend/src/components/FeatureViews/Accounts/TabComponents/ParentContact.js

```diff
@@ -25,26 +25,52 @@ class ParentContact extends Component {
         })
     }
 
+    addDashes(f){
+        return("("+f.slice(0,3)+"-"+f.slice(3,6)+"-"+f.slice(6,15)+")");
+    }
+
     render() {
         //this.state.parent.name
         return (
             <Grid item md={12}>
-            <Grid container spacing={16}>
-                <Grid item xs={3}>
-                        <Paper className={"note"}>
-                            <div className={"actions"}>
-                            {this.state.parent.name}
-                                <EditIcon/>
-                                <AlertIcon/>
-                                <RemoveIcon/>
+                <Grid container spacing={16}>
+                    <Grid item xs={4} >
+                        <Paper className={"ParentContact"}>
+                            <div className="parent-header" align="left">
+                                <Typography className="header-text">
+                                    {this.state.parent.name}
+                                </Typography>
+                            </div>
+                            <div className={"actions"} align="right">
+                                <EditIcon />
                             </div>
                             <Typography className={"body"} align={'left'}>
```

Comment:
you can't have <div> tags under a <p> tag (aka the Typography tag). Make sure you're wrapping <Typography> around the specific text you want. 
![Screen Shot 2019-08-18 at 2 08 18 PM](https://user-images.githubusercontent.com/12959959/63230449-a86bd980-c1c1-11e9-85ba-2883ef0ab996.png)